current/sidebar.json

```diff
@@ -25,7 +25,10 @@
 		{
 			"title": "Integrate",
 			"slug": "integrate",
-			"menu_items": [ { "title": "Tools", "slug": "tools" }, { "title": "Front End", "slug": "front-end" } ]
```

Comment:
a) this has a sidebar?
and b) this change seems unrelated

Reply:
Yes and yes, although I think (a) might be restructured depending on max's work.

Reply:
I just noticed this while I was looking up (a) for myself and it bothered me :)